bin/localnet.sh

```diff
@@ -76,13 +77,19 @@ up)
   fi
   [[ $channel = edge || $channel = beta ]] || usage "Invalid channel: $channel"
 
+  if [[ $2 = -n ]]; then
+    [[ -n $3 ]] || usage "Invalid -n argument"
+    network="$3"
+  fi
+
   (
     set -x
     RUST_LOG=${RUST_LOG:-solana=warn,solana_bpf=info,solana_jsonrpc=info,solana::rpc=info,solana_fullnode=info,solana::drone=info,solana::bank=info,solana::banking_stage=info,solana::system_program=info}
 
     docker run \
       --detach \
       --name solana-localnet \
+      --network "$network" \
```

Comment:
This won't cause a problem when `$network` is empty?

Reply:
No, it actually appears to behave the same as if you don’t specify a network